templates/main.tmpl.php

```diff
@@ -130,7 +130,7 @@
                                 </a> 
                                 <div class="column actions">
                                     <div class="icon-more popupmenu" value="<?php p($poll->getId()); ?>" id="expand_<?php p($poll->getId()); ?>"></div>
-                                    <div class="popovermenu bubble menu" id="expanddiv_<?php p($poll->getId()); ?>">
+                                    <div class="popovermenu bubble menu hidden" id="expanddiv_<?php p($poll->getId()); ?>">
```

Comment:
You need to play with the open class for the popovermenu if you want to fit the css guidelines :)

Reply:
I just added the hidden class, because OC doesn't hide the popover with the used classes. Maybe there is a more elegant way to get this compatible with NC and OC.

Reply:
At least, it works. :-)

Reply:
Maybe with older versions. Master should hide it by default! :)

Reply:
Hmm. That's what it looks like, when I just drop in the app on OC 10.0.3.

![grafik](https://user-images.githubusercontent.com/26707476/31331359-41648da4-ace2-11e7-8a47-6774aa841225.png)